src/store/metadata/stub/content/editions/versions/age-16-to-64-years-level_2023-05.json

```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/age-16-to-64-years-level/editions/2023-05/versions",
```

Comment:
Please effect all the changes requested on the editions file above

Reply:
done